.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+Feature # (details)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).
+
+- [ ] Test A
+- [ ] Test B
```

Comment:
Speaking from personal experience here - would it be useful to add a check of all necessary tests? Namely:
- Unit tests
- E2E locally
- E2E on Github runner
- Potentially E2E on RP but that can't be met by open-source contributors so we can leave that out?

Reply:
The thought here is for the person creating the PR to fill this out with what they have done! I think some of these should be required by default so if we want to add a `required` subsection we can later.

Reply:
makes sense!